recipes/slepc/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set build = 0 %}
+{% set version = '3.7.1' %}
+{% set sha256 = '670216f263e3074b21e0623c01bc0f562fdc0bffcd7bd42dd5d8edbe73a532c2' %}
+{% set mpi = os.environ.get('MPI') or 'mpich' %}
```

Comment:
I think this should either have some explanatory comments, then or leave this for the future.